test/integration/role_test.lua

```diff
@@ -1,11 +1,13 @@
 local fio = require('fio')
 local json = require('json')
 local helpers = require('test.helpers')
-local http_client = require('http.client')
+local http_client = require('http.client'):new()
 local server = require('test.helpers.server')
 
 local t = require('luatest')
-local g = t.group()
+local g = t.group('integration', t.helpers.matrix({
+    use_listen_tls = {true, false}
+}))
```

Comment:
Please, let's avoid it here. It's better to add an endpoint with TLS into the current config + a simple separate test for it.

Reply:
Removed parametrization.